Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -115,6 +154,69 @@ static void CreateConfigFile()
                 EditorPrefs.SetBool(configFileEditorPref, true);
             }
         }
+#if UNITY_EDITOR
+        static void WriteConfigToDisk()
+        {
+            //Check for an already existing persistent config file
+            string fileDirectory = "Assets/LootLockerSDK/Resources/Config/Editor/";
+            string filePath = fileDirectory + "PersistedLootLockerConfig.txt";
+   
+            if(!Directory.Exists(fileDirectory))
+            {
+                Directory.CreateDirectory(fileDirectory);
+                File.Create(filePath).Close();
+            } 
+            else if(!File.Exists(filePath))
+            {
+                File.Create(filePath).Close();
+
+            }
+
+            var tempConfig = Get();
+
+            List<string> config = new List<string>();
+
+            if (!string.IsNullOrEmpty(tempConfig.apiKey))
+            {
+                config.Add(tempConfig.apiKey);
+            }
+
+            if (!string.IsNullOrEmpty(tempConfig.domainKey))
+            {
+                string pattern = @"(?<=https://)\w+(?=\.api\.lootlocker\.io/)";
+                Regex regex = new Regex(pattern);
+                Match match = regex.Match(tempConfig.domainKey);
+
+                if (match.Success)
+                {
+                    string domainkey = match.Value;
+                    Debug.LogWarning("You accidentally used the domain url instead of the domain key,\nWe took the domain key from the url.: " + domainkey);
```

Comment:
I don't think we need to be explicit about this. They should figure it out when the input changes.